docker-compose.yml

```diff
@@ -44,8 +44,6 @@ services:
       # Mainnet
       # NETWORK_ENDPOINT: wss://mainnet-rpc.polymesh.network
       # NETWORK_CHAIN_ID: '0x6fbd74e5e1d0a61d52ccfe9d4adaed16dd3a7caa37c6bc4d0c2fa12e8b2f4063'
-    volumes:
```

Comment:
This was causing issues for some people with file permission issues. Its not needed since the image copies over what it needs when we build it.